### DISCUSSION TITLE
Circle transform / Mesh instancer

### DISCUSSION THREAD

**Discussion by akerb00m** [cables-gl/cables_docs#456] [Answers & Questions] [answered]
Greetings, here a Question about something i used to do with the 'Circle Transform operator' and now wnat to do with the mesh instancer.

I made a setup in cables to illustrate here https://cables.gl/p/IAbMsm

I have been using the 'Route Trigger operator' and the 'fxhash operator' to randomly select a mesh as input for the 'Circle Transform operator' see example -Circle Transform Setup- in the link above.
Since the Circle transform operator is outdated now, i have been trying figure out a way to use the 'Mesh Instancer (and the Circular points operator) but i can't find a way to do it. 

I imagined an operator that has multiple 'geometry' ports on the top, then i attach the GLTFNode Geometry outputs to those ports. Then there could be a number input to select the geometry that gets 'routed' through. (if that makes sense)

Maybe someone Knows of a setup to randomly select a geometry from a 'list' of geometries and input that selection into the mesh instancer operator ?

Cheers !

**Comment by steam0r** [comment, marked as answer]
hey,

if i understand you correctly, these two ops might be of help:

https://cables.gl/op/Ops.Json.RouteObject
https://cables.gl/op/Ops.Json.SwitchObject

best,
stephan/steam

**Comment by akerb00m** [comment]
Heey, the 'switchObject' is perfect, thanks!
gives much better FPS as well : )
Weird that i didn't find that one...
